lib/polyfill/pip_webkit.js

```diff
@@ -47,6 +47,16 @@ shaka.polyfill.PiPWebkit = class {
       return;
     }
 
+    const video =
+      /** @type {!HTMLVideoElement} */ (document.createElement('video'));
+    const isPipPresentationModeSupported =
+      video.webkitSupportsPresentationMode('picture-in-picture');
```

Comment:
We have a constant defined already for this: PiPWebkit.PIP_MODE_

Reply:
Thanks @joeyparrish, see my comment in the initial Par description

> Please note:
> Unable to use static string shaka.polyfill.PiPWebkit.PIP_MODE_ as there's a mix of 
> ES6 features and lack of static class members support so checks fail

Reply:
We run a compiler over our files, so you can use any ES6 features you like.  It may cause problems with uncompiled mode on some platforms, but we don't target uncompiled mode.